components/ConfrimationDialog.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {useTranslation} from 'react-i18next';
+
+import {Dimensions, StyleSheet} from 'react-native';
+import {Overlay} from 'react-native-elements';
+import {Button, Column, Text} from './ui';
+import {Theme} from './ui/styleUtils';
+
+export const ConfirmationDialog: React.FC<ConfirmationDialogProps> = props => {
```

Comment:
typo mistake ConfrimationDialog

Reply:
Why can't we use MessageOverlay and pass two buttons as children

Reply:
deleted confirmation dialog and replaced it with messageOverlay component